src/libs/actions/Link.js

```diff
@@ -38,7 +40,7 @@ function showGrowlIfOffline() {
 function openOldDotLink(url) {
     /**
      * @param {String} [shortLivedAuthToken]
-     * @returns {String}
+     * @returns {Promise <string>}
```

Comment:
Seems like a space was added by accident
```suggestion
     * @returns {Promise<string>}
```